### DISCUSSION TITLE
Option to disable scheduler?

### DISCUSSION THREAD

**Discussion by simonvanderveldt** [cloudera/hue#2352] [Questions] [answered]
Would it be possible to add an option to disable the scheduler?
We already have scheduling provided centrally/outside of Hue, so having another scheduling option is undesired and confusing to the users.

**Comment by romainr** [comment, marked as answer]
```
[desktop]
app_blacklist=oozie

```
?

**Comment by simonvanderveldt** [comment]
@romainr Thanks for the reply, I figured that one out after messing around with the blacklist options and it seems to work, the scheduler is no longer visible. What would be the appropriate place to document this?

Btw I do get a warning in the logs
```
[12/Jul/2021 16:51:45 +0200] api          WARNING  Oozie application is not enabled: Model class oozie.models.Job doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.
```
Is there some code somewhere that should be conditional on oozie being enabled to prevent this?

**Comment by romainr** [comment]
Until the connectors are fully ready,
https://docs.gethue.com/administrator/configuration/server/#disabling-some-apps
?

**Comment by simonvanderveldt** [comment]
OK. The confusion for me mostly came from having to disable "oozie", whereas I expected I'd need to disable "scheduler", was wondering if it would be good to document that.
Or would this discrepancy be solved by the future changes to the connectors?

**Comment by romainr** [comment]
It will indeed be obvious via the new Connector UI
https://docs.gethue.com/administrator/configuration/connectors/
